Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -68,6 +71,28 @@
 
 #define BOARD_NAME         "Printrboard Rev F"
 #define LARGE_FLASH        true
+#ifndef USBCON
+  #error "USBCON should be defined by the platform for this board."
+#endif
+
+// Disable JTAG pins so EXP1 pins work correctly (this header is used for
+// the extrudrboard and filament sensor, for example).
+#define DISABLE_JTAG       true
+
+// Note that REV F6 of the Printrboard stole the A HOTEND pin and
+// reassigned it to a second fan for the extruder heater.  It's
+// recommended that you swap the A and B outputs on the Extrudrboard
+// so EXTRUDERS=2 will still work on F6, using B for E1/HEATER_1/TEMP_1.
+// See https://printrbot.zendesk.com/hc/en-us/articles/115003072346
+
+// If you have REV F6 you probably also want to set E0_AUTO_FAN_PIN
+// to PRINTRBOARD_F6_HEATER_FAN_PIN
+
+// Define NO_EXTRUDRBOARD if you don't have an EXTRUDRBOARD and wish to
+// reassign different functions to EXP1.
+
+// Define NO_EXTRUDRBOARD_OUTPUT_SWAP if you have a REV F5 or lower and
+// want to use EXTRUDRBOARD A for E1 and EXTRUDRBOARD B for E2.
```

Comment:
Wouldn't those #defines belong in your `Configuration.h`?  I tried to avoid any need for an individual builder to edit `pins_PRINTERBOARD_REVF.h`.